Dockerfile.windows.x86_64

```diff
@@ -5,7 +5,7 @@ ARG WIN_VER="ltsc2019"
 FROM mcr.microsoft.com/windows/servercore:$WIN_VER
 
 ENV chocolateyUseWindowsCompression "true"
-ENV RUST_TOOLCHAIN="1.39.0"
+ENV RUST_TOOLCHAIN="1.49.0"
```

Comment:
There is a problem on arm with the 1.49 Rust version: https://github.com/rust-vmm/rust-vmm-container/issues/38

We will need to downgrade it to 1.46. Would it be possible to also build the Windows container with the same version?

Reply:
Sure, it's building right now. I'll update the PR as well to use 1.46.